packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -157,6 +157,33 @@ class ${class_name}:
             result[k - 1] = v
         return Output._make(result)
 
+    def _validate_parameters(self, parameter_values: List[Any]) -> List[Any]:
```

Comment:
I have converted the Path to string here. Passing `Path` to serialization throws error, as serialization in Measurement Plug-In Service excepts string type.